hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -100,6 +100,10 @@ data ServerOutput tx
       , participants :: [OnChainId]
       }
   | DecommitRequested {headId :: HeadId, utxoToDecommit :: UTxOType tx}
+  | DecommitAlreadyInFlight {headId :: HeadId, decommitTx :: tx}
+  | -- | Issued once all parties have signed the 'Snapshot' containing the
```

Comment:
For the reason explained above, I would either remove the haddock here, or point to the specification, or add haddocks everywhere